Update it.m3u for Radio Brianza

marcello1x2 opened the pull request:
This pull request adds Radio Brianza to the Italian playlist.

- Stream: HTTPS and publicly accessible
- Logo: included
- Group: [General / Music / News]

Thank you!

marcello1x2 committed:
Update it.m3u for Radio Brianza TV

marcello1x2 commented:
![radio-brianza-logo-bianco-500](https://github.com/user-attachments/assets/18a24fbe-f822-41d4-9d17-775942d97732)
I add here the logo, probably the latest link doesn't work anymore

freearhey commented on streams/it.m3u:
```diff
@@ -681,3 +681,5 @@ https://rtmp-live-ingest-us-east-1-universe-dacast-com.akamaized.net/transmuxv1/
 http://77.83.120.135:8001/play/a11n
 #EXTINF:-1 tvg-id="Rai1.it@HD",Rai 1 HD (720p)
 http://77.83.120.135:8001/play/a11c
+#EXTINF:-1 tvg-id="radiobrianza.it" tvg-name="Radio Brianza TV", group-title="General",Radio Brianza TV
```
Wrong ID.

https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#how-to-add-a-new-stream-link-to-a-playlists

marcello1x2 committed:
Update it.m3u for Radio Brianza

Removed invalid tvg-id as suggested by reviewer

freearhey commented on streams/it.m3u:
```diff
@@ -681,3 +681,5 @@ https://rtmp-live-ingest-us-east-1-universe-dacast-com.akamaized.net/transmuxv1/
 http://77.83.120.135:8001/play/a11n
 #EXTINF:-1 tvg-id="Rai1.it@HD",Rai 1 HD (720p)
 http://77.83.120.135:8001/play/a11c
+#EXTINF:-1 tvg-id="" tvg-name="Radio Brianza TV", group-title="General",Radio Brianza TV
```
```suggestion
#EXTINF:-1 tvg-id="",Radio Brianza TV
```

> All channel data is taken from the [iptv-org/database](https://github.com/iptv-org/database) repository.

https://github.com/iptv-org/iptv?tab=readme-ov-file#database

marcello1x2 committed:
Update it.m3u

Removed tvg-name and group-title as suggested. Using simplified format based on iptv-org/database structure.

freearhey committed:
Merge branch 'master' into master
